comunicatii/src/main/java/code4romania/czl/scrapers/comunicatii/ScraperComunicatii.java

```diff
@@ -26,216 +26,282 @@
  */
 public class ScraperComunicatii {
```

Comment:
Ar fi de preferat ca scraper-ul sa foloseasca un logger in loc de `System.out`, chiar daca logger-ul ar ajunge sa scrie mesajele in `stdout`.